frontend/src/views/auth/ResetPassword.vue

```diff
@@ -56,9 +56,8 @@
           v-model="confirmation"
           :label="$tc('views.auth.resetPassword.passwordConfirmation')"
           name="Confirmation"
-          vee-id="confirmation"
-          vee-rules="pwConfirmed:@password"
```

Comment:
It was not working on Register.vue (because of a missing validation-observer, which I didn't realize at the time), and I wanted to have it consistent between that page and ResetPassword.vue. I now removed our custom rule `pwConfirmed` completely and returned to the vee-validate built-in `confirmed`.